Source/src/components/ComponentTransform.h

```diff
@@ -38,120 +38,52 @@ namespace Hachiko
         void SetLocalRotation(const Quat& new_rotation_local);
         void SetLocalRotationInEulerAngles(const float3& new_rotation_euler_local);
 
-        static Type GetType();
-        [[nodiscard]] bool HasChanged() const;
-        [[nodiscard]] const float3& GetPosition() const;
-        [[nodiscard]] const float3& GetLocalPosition() const;
-        [[nodiscard]] const float3& GetScale() const;
-        [[nodiscard]] const float3& GetLocalScale() const;
-        [[nodiscard]] const Quat& GetRotation() const;
-        [[nodiscard]] const Quat& GetLocalRotation() const;
-        [[nodiscard]] const float3& GetRotationInEulerAngles() const;
-        [[nodiscard]] const float3& GetLocalRotationInEulerAngles() const;
-        [[nodiscard]] const float3& GetFront() const;
-        [[nodiscard]] const float3& GetUp() const;
-        [[nodiscard]] const float3& GetRight() const;
-        [[nodiscard]] const float4x4& GetMatrix() const;
-        [[nodiscard]] const float4x4& GetLocalMatrix() const;
-       
+        static Type GetType() { return Type::TRANSFORM; };
```

Comment:
In my honest opinion, the previous style of inlined getters looked cleaner on the header file, but this also works if it's better for everyone.